specifications.yml

```diff
@@ -54,7 +54,7 @@ paths:
         required: true
         schema: {}
       responses:
-        200:
+        204:
```

Comment:
204 😳

Reply:
Based on what google tells me, even an empty object counts as 200, 204 is if there is no response at all

Reply:
res.json({}) would be a 200 but res.end() would be a 204 for me I think :thinking: